cmake/Modules/FindComputeCpp.cmake

```diff
@@ -397,19 +396,44 @@ function(add_sycl_to_target)
     "${multi_value_args}"
     ${ARGN}
   )
-  set(fileCounter 0)
-  # Add custom target to run compute++ and generate the integration header
-  foreach(sourceFile ${SDK_ADD_SYCL_SOURCES})
-    if(NOT IS_ABSOLUTE ${sourceFile})
-      set(sourceFile "${CMAKE_CURRENT_SOURCE_DIR}/${sourceFile}")
+
+  # If the CXX compiler is set to compute++ enable the driver. This removes the
+  # need to explicitly generate the sycl integration header in a separate
+  # compiler pass.
+  get_filename_component(cmakeCxxCompilerFileName "${CMAKE_CXX_COMPILER}" NAME)
+  if("${cmakeCxxCompilerFileName}" STREQUAL "compute++")
+    if(MSVC)
```

Comment:
We have a separate CMake file for compiler and platform checks, though I suppose it's no good if you get junk on Windows. What happens? Do you get a sensible error message? Can we use compute++ as a Windows compiler without using sycl-driver?

Reply:
So I have no idea what the state of compute++ on Windows is today with or without the driver. All I know is that apparently when it was tried with Eigen, driver was not working but 2 steps compiler was. I don't have a way of testing on Windows either :/